build/index.js

```diff
@@ -68,9 +68,14 @@ async function main() {
 
     if (
         !existsSync("node_modules") &&
-        ["lint", "lintfix", "clean", "install", "installAndBuild"].includes(
-            command
-        )
+        [
+            "lint",
+            "lintfix",
+            "clean",
+            "build",
+            "install",
+            "installAndBuild",
```

Comment:
There's only a few commands left that _don't_ need the root modules now. I wonder if we should just do this unconditionally? (I'd still favour explicitly rerunning `npm ci` for `install` and `installAndBuild`, but that's a different story :D)

Reply:
I'll do that in another PR, if you don't mind...